tests/cache.test.js

```diff
@@ -122,3 +122,34 @@ describe('With a forced invocation', () => {
         expect(promiseFunc.mock.calls.length).toBe(2);
       }));
 });
+
+describe('With a \'timeToLive\' of \'0\'', () => {
```

Comment:
Maybe we should merge the `debug` branch in first so I can check that the cache stays empty?